### ISSUE TITLE
Default configuration.txt not consistent with documentation

### ISSUE THREAD

**Issue by dark-penguin** [webbukkit/dynmap#2682]
It seems like the documentation ( https://github.com/webbukkit/dynmap/wiki ) lists different parameters than what I got in my default generated configuration.txt . And the Internet tells me there is no way to have Dynmap check its config and complain if any parameters are missing or unrecognized.

In the default configuration.txt which was generated by running Dynmap-3.0-beta-4-forge-1.12.2.jar with no configuration, I have:
```
render-triggers:
  - blockupdate
  #- blockupdate-with-id
  #- lightingupdate
  - chunkpopulate
  - chunkgenerate
  #- none
```
The documentation has many more triggers which are not listed here, as well as chunkgenerate**d**, and no mention of _either_ of those. The Internet tells me those parameters were outdated at least five years ago...

Is it really generating a broken config by default? o_O Curiously, it still seemed to work on my test server before I got to configuring it.

Is there a way to check if any other parameters it autogenerated are actually invalid? Usually, I would trust the autogenerated config over documentation (because documentation can often be outdated, but the program is always consistent with itself).

**Comment by dark-penguin** [comment]
I also see reasons to suspect outdated documentation here:
https://github.com/webbukkit/dynmap/wiki/Support-for-Minecraft-Servers-other-than-CraftBukkit
Those project merged a while ago, and there is no ZIP file to unpack now, right? Or does the Forge version require replacing render-triggers in the documentation with what I had?..

**Comment by dark-penguin** [comment]
Also, there was a number of instances where the actual generated default was different from the in-file documentation. For example,
```
# Enable/disable having the web server allow symbolic links (true=compatible with existing code, false=more secure (default))
allow-symlinks: true
```
Does that mean that by default, the setting is incompatible with the mod itself, or that it was actually supposed to say "default=true"?..

**Comment by dark-penguin** [comment]
OK, I've figured out that when using Dynmap with Forge, the supported render-triggers are different. this really should be in the documentation. I could help updating the documentation, but first, I must find out exactly what those triggers mean - especialy `chunkpopulate`.  I could not find any explanation anywhere. Well, I know it certainly does NOT mean "when players are populating the chunk", because that was removed due to humongous amount of render updates, but that's all I could find.

Are those the only triggers supported by Forge? And is there any documentation about what do they actually mean?

This really looks like a great tool, if not for lack of documentation. I could probably help with that, if I can get someone to explain all that to me in the first place.
